### ISSUE TITLE
[Add Control] CyberComboBox

### ISSUE THREAD

**Issue by Sharaf-Mansour** [Taiizor/ReaLTaiizor#171]
I am building an app with the Cyber Theme, and I noticed that the CyberComboBox does not exist

Is it possible to Customize one to look like this? 
![image](https://github.com/user-attachments/assets/440ce014-b8c6-4aa4-a5ca-4bf9eee77733)
 
Thank you for your time and Consideration. Love the library so far!

**Comment by Taiizor** [comment]
I'm glad you love the library. However, I stopped actively developing the project a long time ago. I only fix small issues from time to time. I believe everyone should transition from WinForms to WPF/UWP because WinForms has become quite outdated. With more modern technologies, it's possible to develop applications with much better aesthetics.

**Comment by Sharaf-Mansour** [comment]
@Taiizor I agree with you 100% I avoid desktop apps myself, and always go Blazor. But some clients want to build launchers type apps and configuration apps which is best fit for Desktop. WinForm for me is easy work. I hate XAML.
Also, the I noticed a lot of Bugs in the CyberTheme like the RadioButton is not even Radio and it behave exactly like a checkbox. I would love to contribute myself one day and keep this alive, But it was really one and done project. But again thank you for clarifying feel free to close this issue.

**Comment by Taiizor** [comment]
> @Taiizor I agree with you 100% I avoid desktop apps myself, and always go Blazor. But some clients want to build launchers type apps and configuration apps which is best fit for Desktop. WinForm for me is easy work. I hate XAML. Also, the I noticed a lot of Bugs in the CyberTheme like the RadioButton is not even Radio and it behave exactly like a checkbox. I would love to contribute myself one day and keep this alive, But it was really one and done project. But again thank you for clarifying feel free to close this issue.

If you enjoy using Blazor, try using MAUI Hybrid (Blazor). You can develop applications for both desktop and mobile just like creating a website. Check out the tutorial [here](https://learn.microsoft.com/aspnet/core/blazor/hybrid/tutorials/maui-blazor-web-app).

**Comment by Taiizor** [comment]
A `CyberComboBox` control has been added in the develop branch and will be included in the next release.

**Features:**
- Full Cyber theme styling: rounded corners, border, RGB animation, lighting/shadow effects, linear gradient borders
- Wraps a native `ComboBox` with Cyber-themed custom painting
- Dropdown arrow indicator drawn in theme accent color (supports RGB mode)
- Exposes key ComboBox properties: `Items`, `SelectedIndex`, `SelectedItem`, `DropDownStyle`, `MaxDropDownItems`
- `SelectedIndexChanged` event
- `StateStyle` support (Default, Custom, Random)
- Consistent API with other Cyber controls (`CyberTextBox`, `CyberButton`, etc.)
